notebooks/triton/aks-gpu-deploy.ipynb

```diff
@@ -0,0 +1,268 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# (PREVIEW) Deploying a web service hosted on NVIDIA Triton to Azure Kubernetes Service (AKS)\n",
+    "This notebook shows the steps for deploying a service with [NVIDIA Triton Inference Server](https://developer.nvidia.com/nvidia-triton-inference-server): registering a model, creating an image, provisioning a cluster (one-time action), and deploying a service to it. \n",
+    "\n",
+    "In this case, we use a Densenet image classification model running with ONNX Runtime, but Triton also supports TensorFlow, PyTorch, and Caffe models.\n",
+    " \n",
+    "We then test and delete the service, image and model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import azureml.core\n",
+    "\n",
+    "print(azureml.core.VERSION)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Get workspace\n",
+    "Load existing workspace from the config file info. If you are running this notebook in a Compute Instance, a configuration file has already been created for you. If you are running this notebook somewhere else, please follow the steps to [create a configuration file](https://docs.microsoft.com/azure/machine-learning/how-to-configure-environment#workspace)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from azureml.core.workspace import Workspace\n",
+    "\n",
+    "ws = Workspace.from_config()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Create compute cluster\n",
+    "\n",
+    "This script creates a compute cluster. Uncomment the first line in the code cell below to see what is happening in that file.\n",
+    "\n",
+    "Note that the setup script assumes you have quota for [NCSv3 machines](https://docs.microsoft.com/azure/virtual-machines/ncv3-series) in the South Central US region. If you need to request additional quota, please create a support request. You can also pass in the --vm_size parameter to setup.py and specify a different VM size.\n",
+    "\n",
+    "Note that this cell can take 10-15 minutes to run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Uncomment the below line to see the contents of setup.py\n",
+    "# %cat ../scripts/setup.py"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "\n",
+    "!python setup.py --compute_loc='westus2' --vm_size='Standard_NC6'"
```

Comment:
can we do this in python code in the cell? not a fan of running a bunch of scripts from jupyter cells